mmdet3d/datasets/convert_utils.py

```diff
@@ -330,7 +330,7 @@ def convert_annos(info: dict, cam_idx: int) -> dict:
         Box3DMode.LIDAR, np.linalg.inv(rect @ lidar2cam0), correct_yaw=True)
     # convert gt_bboxes_3d to cam coordinates
     gt_bboxes_3d = gt_bboxes_3d.convert_to(
-        Box3DMode.CAM, rect @ lidar2cami, correct_yaw=True).tensor.numpy()
```

Comment:
Will this raise an error if gt_bboxes_3d is an instance of `BaseInstance3DBoxes`?

Reply:
This PR will support tensor-like methods (such as `cuda()`, `numpy()`, `detach()` etc.) for `BaseInstance3DBoxes` and `BasePoints`.